src/cryptography/hazmat/bindings/_rust/declarative_asn1.pyi

```diff
@@ -36,6 +38,14 @@ class Encoding:
     Implicit: typing.ClassVar[type]
     Explicit: typing.ClassVar[type]
 
+class Size:
+    min: int
+    max: int | None
+
+    def __new__(cls, min: int, max: int | None) -> Size: ...
```

Comment:
I guess if you want no min you just do min=0?

Reply:
yup